bundles/org.openhab.binding.yamahamusiccast/src/main/java/org/openhab/binding/yamahamusiccast/internal/dto/PlayInfoTuner.java

```diff
@@ -29,10 +28,10 @@ public class PlayInfoTuner {
     private String band;
 
     @SerializedName("fm")
-    private fmObject fm;
+    private FmObject fm;
 
     @SerializedName("dab")
-    private dabObject dab;
+    private DabObject dab;
```

Comment:
doesn't this throw SAT errors?

Reply:
No, there are no SAT errors. Why should there be? 
I'd rather expect the lowercase class name to have thrown a SAT error.